src/base/cvar.mli

```diff
@@ -52,11 +52,11 @@ module type Intf = sig
   val var_indices : t -> int list
 
   val to_constant : t -> field option
-
 end
 
 module Unsafe : sig
   val of_index : int -> 'field t
 end
 
-module Make (Field : Snarky_intf.Field.Extended) : Intf with type field := Field.t
```

Comment:
Can you squash this commit with the previous one? It seems to be only reformating the previous changes.